docs/operate/reference/components/board/esp32.md

```diff
@@ -151,7 +151,7 @@ The following properties are available for `digital_interrupts`:
 ### PWM signals on `esp32` pins
 
 You can set PWM frequencies with Viam through the [`GPIOPin` API](/dev/reference/apis/components/board/#api).
-Refer to the [Espressif documentation for valid frequencies and duty resolutions](https://docs.espressif.com/projects/esp-idf/en/v4.4/esp32/api-reference/peripherals/ledc.html?#supported-range-of-frequency-and-duty-resolutions).
+Refer to the [Espressif documentation for valid frequencies and duty resolutions](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/peripherals/ledc.html#supported-range-of-frequency-and-duty-resolutions).
```

Comment:
I think this should be to the `v5.4` docs: https://docs.espressif.com/projects/esp-idf/en/release-v5.4/esp32/api-reference/peripherals/ledc.html

Reply:
```suggestion
Refer to the [Espressif documentation for valid frequencies and duty resolutions](https://docs.espressif.com/projects/esp-idf/en/release-v5.4/esp32/api-reference/peripherals/ledc.html#supported-range-of-frequency-and-duty-resolutions).
```